lua/dyn_menu.lua

```diff
@@ -16,9 +16,23 @@
 local menu_prop = 'user-data/menu/items'
 local menu_items = mp.get_property_native(menu_prop, {})
 
+local function esc_for_codec(str)
```

Comment:
Do you think we should lowercase `str` before compare? Not sure if mpv will return uppercased codec.

Reply:
No need to do this, mpv provides codec names through ffmpeg, which are all lowercase. You can provide this command of `ffmpeg -codecs` to view the list of codecs supported by the compiled ffmpeg.